travis/docker-compose-build.yaml.mako

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Couldn'e we use the same docker-compose as in scaffold.
I see external db is not in scaffold, is it impossible to add it in scaffold ?

Reply:
@sbrunner: note that it should be possible to use a docker-compose.override.yml file here to add only the external_db service (and do not duplicate the others services configurations).

Reply:
The idea was good, but not sure it possible because of the use of mako.

Reply:
I will investigate it :-)